js-apps/meep-frontend/src/js/containers/exec/mobility-event-pane.js

```diff
@@ -24,19 +25,24 @@ import { EXEC_EVT_MOB_TARGET, EXEC_EVT_MOB_DEST } from '../../meep-constants';
 
 import { getElemFieldVal, FIELD_NAME } from '../../util/elem-utils';
 
+import {
+  uiExecChangeMobilityEventTarget,
+  uiExecChangeMobilityEventDestination
+} from '@/js/state/ui';
+
 class MobilityEventPane extends Component {
   constructor(props) {
     super(props);
-    this.values = {
-      eventType: '',
-      eventTarget: '',
-      eventDestination: ''
-    };
+    // this.values = {
```

Comment:
General comment: No need to keep old code commented out unless it has value. Suggest removing unnecessary comments.

Reply:
Removed the commented code